src/webhooks/pubsub/slackScores.ts

```diff
@@ -198,17 +208,24 @@ export const sendGitHubEngagementMetrics = async (
   await Promise.all(slackNotifications);
 };
 
-export const sendDiscussionMetrics = async (ospo_internal: boolean = false) => {
-  const { discussions, discussionCommenters } = await getDiscussionEvents();
-  if (discussions.length === 0 && discussionCommenters.length === 0) {
+export const sendGitHubActivityMetrics = async (
+  ospo_internal: boolean = false
+) => {
+  const { discussions, githubCommenters, issues } =
```

Comment:
nit: is it `github` or `gitHub`? I'd argue for the latter, but in any case consistency makes the codebase easier to grep.

Reply:
changed to latter for consistency